notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -46,19 +46,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 10,
    "id": "563ecf9f-346b-4f14-85ef-c66ff0c95f65",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "%pip install -q --extra-index-url https://download.pytorch.org/whl/cpu\\\n",
-    "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
-    "\"nncf>=2.6.0\"\\\n",
-    "\"gradio\"\\\n",
-    "\"onnx\" \"onnxruntime\" \"einops\" \"transformers>=4.31.0\"\\\n",
-    "\"openvino==2023.2.0.dev20230922\""
+    "# %pip install -q --extra-index-url https://download.pytorch.org/whl/cpu\\\n",
```

Comment:
Why is requirements installing commented out?

Reply:
i will uncomment this. thanks for noticing

Reply:
Uncommented this.